images/postscript.sh

```diff
@@ -1,5 +1,5 @@
 ## Make sure fstab exists and is empty
-cat > /etc/fstab
+echo -n > /etc/fstab
```

Comment:
sorry this is late, but what's fstab used for? is it related to vsock?

Reply:
This just overwrites fstab with an empty one so that Linux doesn’t mount any drives at boot.